runtime/help/plugins.md

```diff
@@ -157,7 +157,7 @@ The packages and their contents are listed below (in Go type signatures):
     [InfoPane](https://pkg.go.dev/github.com/zyedidia/micro/v2/internal/action#InfoPane)
     [Tab](https://pkg.go.dev/github.com/zyedidia/micro/v2/internal/action#Tab)
     [TabList](https://pkg.go.dev/github.com/zyedidia/micro/v2/internal/action#TabList)
-    [interface{} / any](https://go.dev/tour/methods/14)
+    [any](https://go.dev/tour/methods/14)
```

Comment:
This link https://go.dev/tour/methods/14 still has no mention of the `any` keyword. So to someone who's not aware what `any` is, this link would be confusing (and someone who knows what `any` is doesn't need this link in the first place)?

Reply:
So you'd recommend to stay with `interface{} / any`, right?

Reply:
I would just remove the link: `any` is somewhat self-explanatory (unlike `interface{}`), and it's not specific to micro.

Reply:
> So you'd recommend to stay with `interface{} / any`, right?

Not really, that would be still confusing, right?



> I would just remove the link: `any` is somewhat self-explanatory (unlike `interface{}`), and it's not specific to micro.

Yes, it seems ok to just remove it.